bin/migrate-oats-data/noi/oats_to_alcs_notice_of_intent_table_etl/oats_to_alcs_notice_of_intent_table_etl.py

```diff
@@ -114,6 +115,7 @@ def _get_update_query_from_oats_alr_applications_fields():
                     ag_cap_consultant = COALESCE(%(agri_cap_consultant)s, ag_cap_consultant),
                     ag_cap = COALESCE(%(agri_capability_code)s, ag_cap),
                     legacy_id= COALESCE(%(legacy_application_nbr)s, legacy_id),
+                    proposal_end_date = %(proposal_end_date)s,
```

Comment:
is this field in the list of fields that should not be overridden?

Reply:
does not appear to be in that list